src/Symfony/Bridge/Twig/Extension/HtmlEntitiesExtension.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * Provides a HTML entities filter.
+ *
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+class HtmlEntitiesExtension extends AbstractExtension
```

Comment:
instead of a dedicated exception, can't we make it in an existing one?

Reply:
`FormExtension` sounds good yes, and to really be conflict-safe id even sugges `form_html_entities`

Reply:
lgtm

Reply:
the conflict-safe id is solved by not providing a legacy extension name at all (we don't have BC issues with this extension)

Reply:
but we still need a filter name. `html[_]entities` has higher chance to be already defined in userland, hence `var|form_html_entities`.